spanner/setup.py

```diff
@@ -51,12 +51,14 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.27.0, < 0.28dev',
-    'google-auth >= 1.1.0',
-    'grpcio >= 1.2.0, < 2.0dev',
     'gapic-google-cloud-spanner-v1 >= 0.15.0, < 0.16dev',
-    'gapic-google-cloud-spanner-admin-database-v1 >= 0.15.0, < 0.16dev',
     'gapic-google-cloud-spanner-admin-instance-v1 >= 0.15.0, < 0.16dev',
```

Comment:
Do I presume there is a similar PR coming for the `admin-instance` GAPIC / proto stuff?

Reply:
You assume correctly. Doing them one at a time for review sanity.